axis_utils/include/axis_utils2.inc

```diff
@@ -351,45 +182,46 @@ end subroutine axis_edges
     !        lon =      0 1 2 3 4 5  ...  358 359; lon_strt = 3 ==>
     !        tranlon =  3 4 5 6 7 8  ...  359 360 361 362; istrt = 4
 
-    real, intent(inout), dimension(:) :: lon
-    real, intent(in) :: lon_start
-    integer, intent(out) :: istrt
+    real(kind=FMS_AU_KIND_), intent(inout), dimension(:) :: lon
+    real(kind=FMS_AU_KIND_), intent(in)                  :: lon_start
+    integer, intent(out)                                 :: istrt
 
 
-    integer :: len, i
-    real :: lon_strt, tmp(size(lon(:))-1)
+    integer                 :: len, i
+    real(kind=FMS_AU_KIND_) :: lon_strt, tmp(size(lon(:))-1)
 
     len = size(lon(:))
 
-    do i=1,len
+    do i = 1, len
        lon(i) = lon_in_range(lon(i),lon_start)
     enddo
 
-    istrt=0
-    do i=1,len-1
+    istrt = 0
+    do i = 1,len-1
        if (lon(i+1) < lon(i)) then
-          istrt=i+1
+          istrt = i+1
           exit
        endif
     enddo
 
     if (istrt>1) then ! grid is not monotonic
-       if (abs(lon(len)-lon(1)) < epsln) then
+       if (abs(lon(len)-lon(1)) < real(epsln, FMS_AU_KIND_)) then
           tmp = cshift(lon(1:len-1),istrt-1)
           lon(1:len-1) = tmp
-          lon(len) = lon(1)
+          lon(len)     = lon(1)
        else
           lon = cshift(lon,istrt-1)
        endif
+
        lon_strt = lon(1)
        do i=2,len+1
-          lon(i) = lon_in_range(lon(i),lon_strt)
+          lon(i)   = lon_in_range(lon(i),lon_strt)
           lon_strt = lon(i)
        enddo
     endif
 
     return
-  end subroutine tranlon
+  end subroutine TRANLON_
 
   !>     nearest_index = index of nearest data point within "array" corresponding to
```

Comment:
I think this set of comments is misplaced (and repeated, it is also found for NEAREST_INDEX_).  Could we remove it?